### ISSUE TITLE
Source-map ignores devtoolModuleFilenameTemplate

### ISSUE THREAD

**Issue by gartz** [webpack-contrib/less-loader#83]
![visual ly marketplace 2016-03-22 14-05-22](https://cloud.githubusercontent.com/assets/99608/13967973/3ab6b702-f037-11e5-8d15-7c25f03c1dc1.png)

The less file generated address ins't one defined by `devtoolModuleFilenameTemplate` what makes the browser create 2 domains to acess less source-map.

Config used:

```
output.devtoolModuleFilenameTemplate = 'webpack://marketplace-local.visual.ly.source/[resource-path]?[hash]'
```

**Comment by gartz** [comment]
Related with https://github.com/webpack/less-loader/pull/73 (but it doesn't fully fix this issue, only the part of the incorrect paths).

**Comment by jhnns** [comment]
Thx for reporting this. Just for your information: #73 has been shipped with `2.2.3`

**Comment by gartz** [comment]
That's a great news, now we can proceed and create the fix for the `devtoolModuleFilenameTemplate` support since the file structure is correct :)
